libyul/optimiser/StackToMemoryMover.cpp

```diff
@@ -31,34 +31,34 @@ namespace
 {
 vector<Statement> generateMemoryStore(
 	Dialect const& _dialect,
-	langutil::SourceLocation const& _loc,
+	std::shared_ptr<DebugData const> const& _debugData,
```

Comment:
```suggestion
	shared_ptr<DebugData const> const& _debugData,
```

Also one more below.